doc/developer/testing.rst

```diff
@@ -9,6 +9,83 @@ Quick instructions
 To run the tests, simply execute the following command. Omit ``SR_PYTHON=On`` or ``SR_FORTRAN=On`` if you do not wish to test those languages:
    ``make test SR_PYTHON=On SR_FORTRAN=On``
 
+####################
+Running single tests
+####################
+
+With the increased automation in the SmartRedis test system, it is now more complex to run
+an individual test case. The steps to follow are as follows:
+
+1. Identify the test you wish to run. This is most easily done by running the full test suite
```

Comment:
I would add a line between "Identify the test you wish to run" - will look something like:
Identify the test you wish to run

This is most easily done

Reply:
Sounds good. Did it for the first two items